app/helpers/application_helper.rb

```diff
@@ -9,6 +9,11 @@ def page_title(title = nil)
     end
   end
 
+  def widget_page?
+    ["single_widget", "embed"].include?(controller_name) ||
+      "dashboards_show" == "#{controller_name}_#{action_name}"
```

Comment:
@juliusv made it specifically check for dashboards_show, was easier at this point